larray/core/group.py

```diff
@@ -12,7 +12,7 @@
 from larray.util.oset import *
 from larray.util.misc import basestring, PY2, unique, find_closing_chr, _parse_bound, _seq_summary
 
-__all__ = ['LGroup', 'LSet', 'PGroup', 'union']
+__all__ = ['Group', 'LGroup', 'LSet', 'PGroup', 'union']
```

Comment:
why? Group is supposed to be only a base class right?

Reply:
because I needed to do something like if isinstance(v, Group) in some user code (I don't remember where nor why) and it felt cleaner to just export it than use if isinstance(v, (LGroup, PGroup))

Reply:
OK. Now I understand.